python/python/tests/test_async_client.py

```diff
@@ -1190,6 +1190,33 @@ async def test_sunionstore(self, redis_client: TRedisClient):
             "g",
         }
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_sinter(self, redis_client: TRedisClient):
+        key1 = f"{{testKey}}:{get_random_string(10)}"
+        key2 = f"{{testKey}}:{get_random_string(10)}"
+        non_existing_key = f"{{testKey}}:non_existing_key"
+        member1_list = ["a", "b", "c"]
+        member2_list = ["c", "d", "e"]
+
+        # positive test case
+        assert await redis_client.sadd(key1, member1_list) == 3
+        assert await redis_client.sadd(key2, member2_list) == 3
+        assert await redis_client.sinter([key1, key2]) == {"c"}
```

Comment:
I think such check is better
```suggestion
        assert await redis_client.sinter([key1, key2]) == set(["c"])
```

Reply:
either way will work, I'm curious, why do you think the second way is better? both yield the same result (a set containing "c")